### ISSUE TITLE
Any news on when the new release will happen?

### ISSUE THREAD

**Issue by vazkir** [airbnb/react-dates#2201]
Hi,

It has been 2 years and 5 months and I was wondering when the changes from these past years will be released? I see the repo itself and commits have still been going on, it only doesn't seem that any new release was made

I am asking this since there has been a [bug fix in dec 2021](https://github.com/react-dates/react-dates/pull/1893) that we would really like to see released, since it's now impossible to type into the 1 of the 2 date boxes

Thanks in advance for any clarification!

**Comment by jonlambert** [comment]
FWIW this project has all the hallmarks of abandonware. We're still holding out for React 17 support (released ~2 years ago). 

Personally I'm looking to migrate now.

**Comment by ljharb** [comment]
@jonlambert that's not a useful or accurate characterization.

**Comment by ljharb** [comment]
@vazkir since nobody has helped me maintain this project in those two years, I'm still trying to find the time to cut the next release. I have certainly not forgotten about it.

**Comment by jonlambert** [comment]
Sorry @ljharb - no offense meant. Appreciate the work you've done here. 

Ultimately, with the last release being ~3 years ago and no information on when React 17/18 support will be added, the project had the hallmarks of abandonware. Happy to be corrected on that.

**Comment by ljharb** [comment]
React 17 and 18 support is blocked on enzyme and airbnb-prop-types (which is itself blocked on enzyme) supporting it. I doubt there will be many other obstacles.

**Comment by Flui** [comment]
I think you should better switch from enzyme to @testing-library. Enzyme seems to get no adapter for react 18.

**Comment by ljharb** [comment]
@Flui RTL is not equivalent to enzyme and lacks many of its capabilities. Indeed enzyme has no adapter for react 17 or 18 yet (i'm the sole maintainer of enzyme).

**Comment by ThiefMaster** [comment]
But what's the solution here? Will react-dates remain stuck with an outdated react version nearly forever since react 18 support in enzyme doesn't seem to be happening anytime soon?

I guess I'm not the only user of react-dates who couldn't care much less about testing. Sure, it's nice to have good test coverage, but I'd rather have a release with less test coverage than having to stick with React 17 for another year or two...

**Comment by ljharb** [comment]
Until enzyme supports it, yes. Which won’t be forever.

**Comment by ThiefMaster** [comment]
@ljharb Could you provide at least a rough timeline? Are we weeks, months or possibly years away from this?

**Comment by ljharb** [comment]
@ThiefMaster i'm not really sure. I've been waiting for an employer, or sponsors, to fund the work for about 7 years now (or to find time myself), and altho tons of people have complained, and have spent person-hours migrating their codebase off of enzyme, zero people seem to have provided sufficient funding to warrant me being able to devote sufficient time to complete the work. This could have been done in 2020, let alone ASAP, if one or more folks is willing to invest in my time.

**Comment by ThiefMaster** [comment]
What amount of money are we talking about? I think with a price tag on it people would be more likely to consider this. ;)

**Comment by ljharb** [comment]
I think a month worth of person-hours, whether just me or spread across myself and other developers, would be more than sufficient. I remain as willing now as I've been for the last half decade to discuss it privately :-)
